lib/utils/getTreasuryBalance/index.ts

```diff
@@ -48,9 +48,8 @@ export const getTreasuryBalance = async (
     const nakedBCT = getInteger(
       await bctContract.balanceOf(addresses.mainnet.treasury)
     );
-    const bctUSDC = await getOwnedBCTFromSLP({ adr: "bctUsdcLp", provider });
     const bctKLIMA = await getOwnedBCTFromSLP({ adr: "klimaBctLp", provider });
-    const sum = nakedBCT + bctUSDC + bctKLIMA;
```

Comment:
ohhhh this is one we didn't think about. We need to keep it, as our treasury still has LP and we want that shown in our total tonnes on the landing page.

I believe all you have to do is revert this line then add back the address in `lib/constants` and everything should work again